docker-for-windows/index.md

```diff
@@ -478,15 +478,15 @@ You can configure options on the Docker daemon in the given JSON configuration f
 
 ![Docker Daemon](images/docker-daemon.png)
 
-For a full list of options on the Docker daemon, see <a href="https://docs.docker.com/engine/reference/commandline/dockerd/" target="_blank">daemon</a> in the Docker Engine command line reference.
+For a full list of options on the Docker daemon, see [daemon](/engine/reference/commandline/dockerd/) in the Docker Engine command line reference.
```

Comment:
I can use this syntax with Kramdown: `[link](url){:target="_blank"}`
I'll fix it now.